helper.go

```diff
@@ -48,6 +49,56 @@ func ImageExists(filename string) bool {
 	return !info.IsDir()
 }
 
+// Check for remote filepath, e.g: https://test.webp.sh/node.png
+// return StatusCode, etagValue
+func GetRemoteImageInfo(fileUrl string) (int, string) {
+	res, err := http.Head(fileUrl)
+	if err != nil {
+		log.Fatal("Connection to remote error!")
+	}
+	if res.StatusCode != 404 {
+		for index, _ := range res.Header {
+			if strings.ToLower(index) == "etag" {
```

Comment:
By the way, `res.Header.Get()` is case insensitive, that means `res.Header.Get("cache-controL")` and `res.Header.Get("cache-Control")` are exactly the same